### PR TITLE
Rework of PR #69 to fix Issue #68

### PR DESCRIPTION
**Pull request by ixs**
Issue #68 raised a valid problem: The postfix formula is using
the osmap.yaml file incorrectly. It is being used to filter on
the os grain but should instead be filtering on os_family
to correctly account for Red Hat/CentOS similarities.

PR #69 did the fix in a complex way as it only moved parts of the os
mapping into the os_family mapping.

This is a much simpler fix, as it just renames the file and uses
it as a os_family map, which it actually is.

If we really need to differentiate between os flavors inside the
os_family at a later date, we can then add a lookup on the os grain.
No need to complicate manners now.

### CODE REVIEW

**Comment by ixs** [comment]
@javierbertoli done.

File: postfix/map.jinja [hunk 1]
```diff
@@ -2,13 +2,12 @@
 # vim: ft=jinja
 
 {% import_yaml "postfix/defaults.yaml" as defaults %}
-{% import_yaml "postfix/osmap.yaml" as osmap %}
+{% import_yaml "postfix/os_familymap.yaml" as os_familymap %}
```
Comment (javierbertoli) [inline postfix/map.jinja]: You forgot to change it here

Comment (ixs) [reply postfix/map.jinja]: fsck. fixed.

**Comment by javierbertoli** [comment]
Thanks @ixs!

**Pull request merged**